src/components/v5/common/ActionSidebar/hooks/permissions/useHasNoDecisionMethods.ts

```diff
@@ -33,6 +42,25 @@ const useHasNoDecisionMethods = () => {
     return false;
   }
 
+  const requiredRolesDomain = getPermissionsDomainIdForAction(actionType, {});
+
+  // If the requiredRolesDomain is root, check the user has the required permissions in root
+  if (requiredRolesDomain === Id.RootDomain) {
+    const userRootRoles = getUserRolesForDomain(
+      colony,
+      user.walletAddress,
+      Id.RootDomain,
+    );
+
+    if (
+      !requiredPermissions.every((requiredPermission) =>
+        userRootRoles.includes(requiredPermission),
+      )
+    ) {
+      return true;
+    }
+  }
+
```

Comment:
Could those 2 checks (this and the `getAllUserRoles` below) be combined? `requiredRolesDomain` will be either `Id.RootDomain` or `undefined` (as it might depend on the form values which we don't know at this point) so it should hopefully be pretty straighforward